### PR TITLE
2020.10 set

### PR DESCRIPTION
**Pull request by bedroge**
Work in progress proposal for a `2020.10` set, related to the discussion here:
https://github.com/EESSI/compatibility-layer/issues/47

Note that I've removed `lua` and `tcl` from our set, since `Lmod` now has a dependency on them.

### CODE REVIEW

File: etc/portage/sets/2020.10 [hunk 1]
```diff
@@ -3,3 +3,5 @@ app-editor/vim
 dev-util/strace
 app-admin/Lmod
 sys-apps/archspec
+sys-cluster/rdma-core
+sys-fabric/opa-psm2
```
Comment (boegel) [inline etc/portage/sets/2020.10:7]: @bedroge What about `librdmacm`, don't we need that too? Or does `rdma-core` replace that?

Comment (bedroge) [reply etc/portage/sets/2020.10:7]: Yep, that's included, see https://github.com/linux-rdma/rdma-core.

**Comment by boegel** [comment]
So this is no longer WIP then?

@peterstol Can you take a look?

**Review by peterstol — APPROVED** [review]
Yeah, vim added :-P

**Pull request merged**